src/planner/Query.h

```diff
@@ -1112,6 +1112,43 @@ class UnionAllVersionVar final : public SingleInputNode {
         : SingleInputNode(qctx, Kind::kUnionAllVersionVar, input) {}
 };
 
+// merge columns to one
+// require the type of columns are same
+class ColumnsMerge final : public SingleInputNode {
```

Comment:
Why do you introduce this plan node ? Could project not do what you want ?

Reply:
Project can't merge the different columns to one column.